ui/analyse/src/autoShape.ts

```diff
@@ -107,9 +107,20 @@ export function compute(ctrl: AnalyseCtrl): DrawShape[] {
       const glyph = glyphs[0];
       const svg = glyphToSvg[glyph.symbol];
       if (svg) {
+        const pos = ctrl.position(ctrl.tree.parentNode(ctrl.path)).unwrap();
```

Comment:
this probably adds some overhead, parsing FEN and creating a chessops game.

Can we maybe do it only when the move is a castle?

Reply:
I used `san` version of the move instead. 👍 Possibly undefined (like uci) so in that case it won't show anything (but a move made in a game which has no uci is probably a bug anyway).